src/main/java/org/folio/rest/impl/EholdingsPackagesImpl.java

```diff
@@ -505,20 +500,14 @@ private String getResourceId(TitleResult titleResult) {
       packagePutBodyValidator.validate(entity);
       packagePutBody = converter.convertToRMAPIPackagePutRequest(entity);
     }
-    return context.getPackagesService().updatePackage(parsedPackageId, packagePutBody);
+    return context.getPackagesService().updatePackage(parsePackageId(originalPackage.getFullPackageId()), packagePutBody);
   }
 
-  private boolean isPackageUpdatable(PackagePutRequest entity) {
```

Comment:
I think we should have this validation also. Because we can't update not selected packages that are not custom.